noir-projects/noir-protocol-circuits/crates/rollup-lib/src/base/base_rollup_inputs.nr

```diff
@@ -342,6 +344,22 @@ impl BaseRollupInputs {
         U256::from_bytes32(sha_digest).to_u128_limbs()
     }
 
+    // Computes the out hash for a base rollup
+    fn components_compute_kernel_out_hash(combined: CombinedAccumulatedData) -> [Field; NUM_FIELDS_PER_SHA256] {
+        let mut out_hash_inputs: [Field; MAX_NEW_L2_TO_L1_MSGS_PER_TX] = combined.new_l2_to_l1_msgs;
+
+        let mut hash_input_flattened = [0; CALL_DATA_HASH_FULL_FIELDS * 32];
```

Comment:
The value above is way to large, you are creating something for 197 fields, but you only have a couple of messages so better only create output that we need, otherwise we waste resources and the hash will also be very different from what we expect since it would now be having a whole bunch of zero values at the end. 

You only need field for every input you got here, so `MAX_ NEW_L2_TO_L1_MSGS_PER_TX` should do